modules/@apostrophecms/schema/index.js

```diff
@@ -526,23 +526,21 @@ module.exports = {
             continue;
           }
 
-          if (val.min && destinationKey < val.min) {
-            return false;
-          }
-          if (val.max && destinationKey > val.max) {
-            return false;
+          if (val.min || val.max) {
```

Comment:
0 can be a legitimate value for min or max. These need to be tested for existence.

Reply:
Yep, good idea. Done